CMakeLists.txt

```diff
@@ -95,27 +95,25 @@ if (GOOGLE_CLOUD_CPP_ENABLE_CCACHE)
 endif ()
 
 # Search for Doxygen and create the targets for it if found.
-option(GOOGLE_CLOUD_CPP_GENERATE_DOXYGEN "Generate Doxygen documentation" OFF)
+option(
+    GOOGLE_CLOUD_CPP_GENERATE_DOXYGEN
+    "Generate Doxygen documentation. Only google-cloud-cpp developers are expected to use this flag."
+    OFF)
 mark_as_advanced(GOOGLE_CLOUD_CPP_GENERATE_DOXYGEN)
 
+option(
+    GOOGLE_CLOUD_CPP_USE_LATEST_FOR_REFDOC_LINKS
+    "Link to \"latest\" in our reference documentation. Otherwise, link to the project version. Only google-cloud-cpp developers are expected to use this flag."
```

Comment:
Can we do anything about the line length?

Reply:
Done.

```
$ cmake -LAH

<omitted>

// Generate Doxygen documentation. Only google-cloud-cpp developers are expected to
use this flag.
GOOGLE_CLOUD_CPP_GENERATE_DOXYGEN:BOOL=OFF

<omitted>

// Link to "latest" in our reference documentation. Otherwise, link to the project
version. Only google-cloud-cpp developers are expected to use this flag.
GOOGLE_CLOUD_CPP_USE_LATEST_FOR_REFDOC_LINKS:BOOL=OFF

<omitted>
```

No I don't care that the comments are weirdly stylized. It is obvious to me what is a comment and what is an option. Also, the problem is not unique to these two options.